test/integration/orders.test.js

```diff
@@ -26,16 +26,16 @@ describe('Orders Integration', function () {
     expect(data.price_cents_usd + data.patch_fee_cents_usd).to.eq(100);
   });
 
-  it('supports creating an order with issuedTo', async function () {
-    const issuedTo = { email: 'issuee@companyc.com', name: 'Bob Dylan' };
+  it('supports creating an order with issued_to', async function () {
+    const issued_to = { email: 'issuee@companyc.com', name: 'Bob Dylan' };
     const { data } = await patch.orders.createOrder({
       total_price_cents_usd: 100,
-      issuedTo: issuedTo
+      issued_to: issued_to
```

Comment:
was this a change we made to the API or SDK? Curious how this test was passing / working before

Reply:
I'm also worried about this working previously. After generating the new code, this stopped working. But now that I remember, there was a support issue raised in Discord about this a while back: https://discord.com/channels/733029448558837792/754039768764907550/988853938356891718

Reply:
I'm curious what changed this though. When I generated the code, it looked like the generator had turned the `issued_to` from the API into `issuedTo` for the SDK: https://github.com/patch-technology/patch-node/blob/main/src/model/CreateOrderRequest.js#L74

Can you create a PR for your changes to client-code-generation, I'd like to see what changed there.

Reply:
Now that I look at it more closely though:

- `issuedTo` is the only parameter in camelCase, which isn't great
- but changing to `issued_to` would be a breaking change (e.g. for the person you answered to on [Discord](https://discord.com/channels/733029448558837792/754039768764907550/988853938356891718)

What's surprising to me is why the generation went with `issuedTo` for this one, and not the others 🤔